test/hisi_zip_test/test_sva_perf.c

```diff
@@ -732,6 +733,54 @@ static void handle_sigbus(int sig)
 	        _exit(0);
 }
 
+static int test_sw_dfl_sw_ifl(void)
+{
+	struct hizip_test_info info = {0};
+	struct test_options opts = {
+		.alg_type		= WD_ZLIB,
+		.sync_mode		= 0,
+		.thread_num		= 16,
+		.block_size		= 8192,
+		.total_len		= 8192 * 10,
+		.compact_run_num	= 1000,
+	};
```

Comment:
why not reuse the common input parameter? this breaks the whole test into two parts...